### PR TITLE
👽 Use .formatted instead of format_error

### PR DESCRIPTION
**Pull request by nikolaik**
This stops using the deprecated format_error removed in graphql-core v3.2 https://github.com/graphql-python/graphql-core/commit/09ff14f68bdce1e9cd71decc871fd38274b01971 . `GraphqlError.formatted` has been a thing since at least v1.

This PR tries to solve the same thing as #1326 and #1320.

Looks like there a missing newline in test output. Not sure if to update the tests or the code.

### CODE REVIEW

**Comment by erikwrede** [comment]
Hey there, if someone is able to fix the test I can merge and push a release :)

**Review by erikwrede — CHANGES_REQUESTED** [review]
See my comment; once tests pass I'll merge if you tag me :)

**Review by erikwrede — APPROVED** [review]
LGTM

**Comment by nikolaik** [comment]
@erikwrede thanks! ❤

**Comment by erikwrede** [comment]
Thank you! 🚀

**Pull request merged**
